test/date_range_test.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'test_helper'
+
+describe 'DateRange class' do 
+  before do 
+    start_date = Date.new(2021, 2, 24)
+    end_date = Date.new(2021, 2, 26)
```

Comment:
2021 is pretty far in the future.  If you want to make sure your tests continue to work indefinitely though you can always create `Date`s using `Date.today` and adding some days:

```suggestion
    start_date = Date.today + 24
    end_date = Date.today + 26
```